test/builders/wizards-v1/universe/universe-erc721-warper-builder-v1.ts

```diff
@@ -0,0 +1,131 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { BigNumberish, BytesLike, ContractTransaction } from 'ethers';
+import { ERC721_WARPER_PRESET_IDS } from '../../../../src/protocol/warper/v1-controller/ERC721/constants';
+import { ADDRESS_ZERO } from '../../../../src/utils/bytes-and-hashing';
+import { ITaxTermsRegistry, IUniverseRegistry, IUniverseWizardV1, IWarperManager } from '../../../../typechain';
+import { UniverseConstructorV1 } from '../../../constructors/universe/universe-constructor-v1';
+import { WarperConstructorV1 } from '../../../constructors/warper/warper-constructor-v1';
+import { COMMON_TEST_WARPER_NAME_1 } from '../../../utils/common-test-stuff/dummy-data/universe';
+import { SomethingNotConfiguredError } from '../../../utils/common-test-stuff/general-utils';
+import { BuilderStorageBase } from '../../builder-storage-base';
+
+export class UniverseWithErc721WarperBuilderV1 extends BuilderStorageBase {
+  private universeConstructor?: UniverseConstructorV1;
+  private warperConstructor?: WarperConstructorV1;
+  private taxTerms?: ITaxTermsRegistry.TaxTermsStruct;
+  private warperAddress = ADDRESS_ZERO;
```

Comment:
I am using `WarperConstructorV1 ` because it is more general and can be used by any constructor which extends `WarperConstructorV1`. In my case, I was using `Erc721ConfigurableWarperConstructor` with this builder.

Reply:
WarperRegistrationConstructorV1 is also general! Take a look at it 👍